### ISSUE TITLE
SSL/TLS on macOS appears to be broken

### ISSUE THREAD

**Issue by wfraser** [CDrummond/cantata#1015]
I'm attempting to use Cantana with a remote MPD configured to stream audio over HTTPS (nginx is the actual server, forwarding to mpd). I do see a (brief) connection in nginx's logs, but I don't get any audio output, and enabling the relevant logging I get this:

```
> CANTANA_DEBUG=524288 ./cantata
PasteBoard: Error creating pasteboard: com.apple.pasteboard.clipboard [-4960]
PasteBoard: Error creating pasteboard: com.apple.pasteboard.find [-4960]
qt.network.ssl: QSslSocket: cannot resolve TLSv1_1_client_method
qt.network.ssl: QSslSocket: cannot resolve TLSv1_2_client_method
qt.network.ssl: QSslSocket: cannot resolve TLSv1_1_server_method
qt.network.ssl: QSslSocket: cannot resolve TLSv1_2_server_method
qt.network.ssl: QSslSocket: cannot resolve SSL_select_next_proto
qt.network.ssl: QSslSocket: cannot resolve SSL_CTX_set_next_proto_select_cb
qt.network.ssl: QSslSocket: cannot resolve SSL_get0_next_proto_negotiated
qt.network.ssl: QSslSocket: cannot call unresolved function SSL_get0_next_proto_negotiated
```

If I switch to plain HTTP it works fine (though I still get those errors in the log for some reason).

I'm on macOS 10.12.4 (Sierra). This is using the v2.0.1 official release DMG.

**Comment by CDrummond** [comment]
Yeah, the SSL libraries are missing. I bundle then with the Windows installer, guess I need to do the same for macOS.

**Comment by CDrummond** [comment]
Fixed in git/master for 2.1.0
